backend/main/server/controllers.go

```diff
@@ -359,6 +359,12 @@ func (a *App) createCommunity(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+	if payload.Proposal_threshold != nil && payload.Only_authors_to_submit != nil {
+		err = validateProposalThreshold(*payload.Proposal_threshold, *payload.Only_authors_to_submit)
+		if err != nil {
+			respondWithError(w, http.StatusBadRequest, err.Error())
+		}
+	}
```

Comment:
So I wanted to move as much logic into the helper function as I could by passing the entire payload into the function and then validating but because the shape of the payload for create and update are different it didn't quite work out the way I wanted it to, I found that validating the nil pointer ref and then passing the values was the cleanest way to do so without writing separate functions like `validateCreateCommunityProposalThreshold` and `validateUpdateCommunityProposalTheshold` which I guess an argument could be made for that approach, however comparatively I prefer this way better.  I'm open to a cleaner pattern here as well, I have been looking around for cleaner patterns around nil pointer checking in go as I am still kinda relatively new to the language so i'm open to any suggestions!